src/style.css

```diff
@@ -1 +1,80 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  --main-color: #00acdc;
+}
+
+.link {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+  position: relative;
+  padding: 22.5px 0;
```

Comment:
use integers

Reply:
Decided to use padding: 22.5px on Q&A session.